src/relentless/simulate/hoomd.py

```diff
@@ -70,8 +70,8 @@ def _call_v3(self, sim):
         # parse masses by type
         snap = sim["engine"]["_hoomd"].state.get_snapshot()
         sim.masses = self._get_masses_from_snapshot(sim, snap)
+        sim[self]["_bonds"] = self._get_bonds_from_snapshot(sim, snap)
         self._assert_dimension_safe(sim, snap)
-
         # create the potentials, defer attaching until later
         neighbor_list = hoomd.md.nlist.Tree(buffer=sim.potentials.pair.neighbor_buffer)
```

Comment:
One thing we do still want to do is make sure we can specify the neighbor list exclusions, and then only do the exclusions in the thermo if they are active. I think we talked about adding that as an argument / property of the `PairPotentialTabulator`, like the `neighbor_buffer`, and then we would set it here by setting the `exclusions` option.